ui/packages/consul-ui/tests/acceptance/steps/api-prefix-steps.js

```diff
@@ -0,0 +1,11 @@
+import steps from './steps';
+
+// step definitions that are shared between features should be moved to the
+// tests/acceptance/steps/steps.js file
+
+export default function(assert) {
```

Comment:
Do we need this file?

Reply:
Yeah, every feature file MUST have a corresponding one of these due to the ember addon we use, it's annoying I know, a while back I used something different to get around this but it never made it in.